templates/slurm.txt

```diff
@@ -688,3 +689,13 @@ Order = 20
         Config.Plugin = pico.form.Dropdown
         Config.Entries := {[Value=1], [Value=2], [Value=3], [Value=undefined; Label="Any"]}
         Conditions.Hidden := configuration_slurm_ha_enabled isnt true
+
+        [[parameters Node Healthchecks]]
+        Description = "Section for configuring Node Healthchecks"
+        Order = 12
+
+            [[[parameter EnableNodeHealthChecks]]]
+            Label = Enable NHC tests
+            DefaultValue = false
+            Widget.Plugin = pico.form.BooleanCheckBox
+            Widget.Label = Run Node Healthchecks on startup
```

Comment:
Same as the other one - probably should change to "Run Node Health Checks on startup".